src/client/components/dimension-tile/dimension-tile.tsx

```diff
@@ -640,16 +658,15 @@ export class DimensionTile extends React.Component<DimensionTileProps, Dimension
       showSearch={showSearch}
       icons={icons}
       className={className}
-      actions={actions}
-    >
+      actions={actions}>
       <div className="rows">
         {rows}
         {message}
       </div>
       {foldControl}
-      {error ? <QueryError error={error} /> : null}
-      {loading ? <Loader /> : null}
+      {error && <QueryError error={error} />}
+      {!sortOn && <ErrorMessage message="No measure selected"/>}
```

Comment:
It should be a notice not error :)

Reply:
I don't think there's any difference. It's presentation component, beside displaying error/message/notice/warning, it doesn't have any semantics. If it's Notice, then QueryError could use it?